manual/source/examples/h5py/simple_example_write1/index.rst

```diff
@@ -82,15 +82,16 @@ the information density provided by *punx tree*.
 .. [#] *punx tree* : https://punx.readthedocs.io/en/latest/source_code/h5tree.html#how-to-use-h5tree
 
 downloads
-*********
+=========
```

Comment:
Ah, just a choice of a different header-defining symbol.

Reply:
Indeed, just making header symbols consistent with other files.